(DOCSP-29270): Enterprise Authentication page (#79)
mongodb/docs-kotlin: source/fundamentals/enterprise-auth.txt

>       Your code to instantiate a ``MongoClient`` should look something like this:

Reviewer: ```suggestion
      Your code to instantiate a ``MongoClient`` should resemble the following:
```

Author: Updated throughout

> 

Reviewer: cant comment there but on line 264 i think it looks better to italicize everything including the parentheses. Applies in another place as well

Author: Updated both
